modules/mpi/tutorials/ospMPIDistribTutorialVolumeHDF5.cpp

```diff
@@ -90,13 +90,61 @@ int main(int argc, char **argv)
         },
         nullptr);
 
+    std::cerr << "ABOUT TO READ" << std::endl;
+
     // open the hdf5 file, read the data and create the ospray Volume object
     ChomboHDF5::Reader hdf5reader(argv[1], mpiRank, mpiWorldSize, argv[2]);
+    std::cerr << "READ!" << std::endl;
+
+    VolumeBrick brick;
+
+    //convert from int to float as required by ospray
+    //get the world bounds while we are at it
+    std::vector<box3f> myregions;
+    vec3f min{MAXFLOAT};
+    vec3f max{-MAXFLOAT};
+    for (auto x : hdf5reader.m_blockBounds) {
+        std::cerr << x.lower << " " << x.upper << std::endl;
+        if (x.lower.x < min.x)
+            min.x = x.lower.x;
+        if (x.lower.y < min.y)
+            min.y = x.lower.y;
+        if (x.lower.z < min.z)
+            min.z = x.lower.z;
+        if (x.lower.x > max.x)
+            max.x = x.lower.x;
+        if (x.lower.y > max.y)
+            max.y = x.lower.y;
+        if (x.lower.z > max.z)
+            max.z = x.lower.z;
```

Comment:
It's possible this is quite right since the `int` indices correspond to counting cells (e.g. the lower corner of the cell) and presumably the float indices should correspond to the corners of the whole box so you need to add the cell width to each component of `max`.

Reply:
Any chance a similar mistake like this in the original gravity spheres example was causing your artifacts at the block boundaries you observed?

Reply:
See comment below.

Reply:
Could very well be one of the issues in the GSexample, I'll take a look shortly.